.github/workflows/ci.yml

```diff
@@ -1,30 +1,127 @@
----
```

Comment:
Any reason you didn't like this?

Reply:
I think the document start marker should only exist if it's being used or if there are directives above the marker. Since this is a single document with no directives, it's not adding any value.